sycl/include/CL/sycl/backend_types.hpp

```diff
@@ -24,16 +24,17 @@ enum class backend : char {
   ext_oneapi_level_zero = 2,
   level_zero __SYCL2020_DEPRECATED("use 'ext_oneapi_level_zero' instead") =
       ext_oneapi_level_zero,
-  cuda = 3,
+  ext_oneapi_cuda = 3,
+  cuda __SYCL2020_DEPRECATED("use 'ext_oneapi_cuda' instead") = ext_oneapi_cuda,
   all = 4,
   ext_intel_esimd_emulator = 5,
   esimd_cpu __SYCL2020_DEPRECATED("use 'ext_oneapi_esimd_emulator' instead") =
       ext_intel_esimd_emulator,
-  hip = 6,
+  ext_oneapi_hip = 6,
+  hip __SYCL2020_DEPRECATED("use 'ext_oneapi_esimd_emulator' instead") =
```

Comment:
```suggestion
  hip __SYCL2020_DEPRECATED("use 'ext_oneapi_hip' instead") =
```

I assume this is a typo.

Reply:
@gmlueck ups, sorry. Thank you for the suggestion! @againull has already committed it and I've fixed a linter warning about formatting.